src/Camelot.Services.Abstractions/IShellIconsService.cs

```diff
@@ -0,0 +1,25 @@
+using Camelot.Services.Abstractions.Models;
+
+
+namespace Camelot.Services.Abstractions;
+
+public interface IShellIconsService
```

Comment:
These class & interface should be merged with `IShellService` with some methods being private

Reply:
see notes before about IShellIconsService as per platform.